### PR TITLE
Bugfix for #102 Space should not trigger auto completion of variables

### PR DESCRIPTION
**Pull request by HuanhuanSunMSFT**
@AndreSayreMSFT @EricMSFT @Microsoft/poshtools

### CODE REVIEW

File: PowerShellTools/Intellisense/IntelliSenseManager.cs [hunk 1]
```diff
@@ -79,60 +79,84 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdId, uint nCmdexecopt, IntPtr pv
...
+                        TriggerCompletion();
+                    }
                     return VSConstants.S_OK;
-                }
+                default:
+                    break;
+                    
             }
-            // check for Ctrl-Space usage
-            if (commandId == (uint)VSConstants.VSStd2KCmdID.COMPLETEWORD)
```
Comment (ASayre) [inline PowerShellTools/Intellisense/IntelliSenseManager.cs:122]: Do we still check for ctrl-space elsewhere?

Comment (HuanhuanSunMSFT) [inline PowerShellTools/Intellisense/IntelliSenseManager.cs:122]: Sure, I refactored this to switch-case. With so many conditions check, I prefer switch, which is much cleaner.

Comment (ASayre) [inline PowerShellTools/Intellisense/IntelliSenseManager.cs:122]: Ah, I see where I missed stuff :)

**Comment by ASayre** [comment]
Looks good

**Pull request merged**